src/Cli/func/Actions/AzureActions/PublishFunctionAppAction.cs

```diff
@@ -824,6 +824,12 @@ private async Task<bool> HandleFlexConsumptionPublish(Site functionApp, Func<Tas
 
             if (PublishBuildOption == BuildOption.Remote)
             {
+                if (GlobalCoreToolsSettings.CurrentWorkerRuntime == WorkerRuntime.Python && functionApp.FunctionAppConfig?.Runtime?.Version == "3.14")
+                {
+                    // Remote build for Python 3.14 is pending for Flex consumption
+                    ColoredConsole.WriteLine(WarningColor("Remote build for Python 3.14 is not yet supported for Flex. Please refer to aka.ms/py314-remote-build-flex for more details."));
```

Comment:
So this is a warning rather than error, do you want the publish action to stop? If so we should throw instead?